src/server/src/test/java/io/cassandrareaper/service/RepairRunnerTest.java

```diff
@@ -958,16 +935,51 @@ protected JmxProxy connectImpl(Node host) throws ReaperException {
     storage.updateRepairRun(
         run.with().runState(RepairRun.RunState.RUNNING).startTime(DateTime.now()).build(runId));
     // We'll now change the list of replicas for any segment, making the stored ones obsolete
+    when(clusterFacade.getRangeToEndpointMap(any(), anyString()))
+        .thenReturn((Map)ImmutableMap.of(
+            Lists.newArrayList("0", "100"), Lists.newArrayList(nodeSetAfterTopologyChange),
+            Lists.newArrayList("100", "200"), Lists.newArrayList(nodeSetAfterTopologyChange)));
+    when(clusterFacade.getEndpointToHostId(any())).thenReturn(nodeMapAfterTopologyChange);
     when(clusterFacade.tokenRangeToEndpoint(any(), anyString(), any()))
       .thenReturn(Lists.newArrayList(nodeSetAfterTopologyChange));
     context.repairManager.resumeRunningRepairRuns();
     // The repair should now fail as the list of replicas for the segments are different from storage
     await().with().atMost(20, TimeUnit.SECONDS).until(() -> {
```

Comment:
I'm having a hard time trying to understand the intent here. This Awaitility call always fails, throws a `ConditionTimeoutException`, which the test expects to receive, and the rest of the test is basically dead code. Could you refactor this to better express the intent? Afaict the intent seems to be: "wait 20 seconds and then assert that the run state is not ERROR". I wonder if we couldn't simplify this to "await at most 20 seconds until the run state is DONE", wdyt?

Also, is this comment still relevant?

> The repair should now fail as the list of replicas for the segments are different from storage

Afaict the repair is not failing.